src/erlcloud_sns.erl

```diff
@@ -192,28 +194,28 @@ confirm_subscription2(Token, TopicArn, Config) ->
     erlcloud_xml:get_text(
         "ConfirmSubscriptionResult/SubscriptionArn", Doc).
 
--spec confirm_subscription2/4 :: (string(), string(), string(), string()) -> string().
+-spec confirm_subscription2/4 :: (parameter(), parameter(), parameter(), parameter()) -> string().
```

Comment:
Line 197 is too long: -spec confirm_subscription2/4 :: (parameter(), parameter(), parameter(), parameter()) -> string()..